src/main/java/ch/njol/skript/doc/HTMLGenerator.java

```diff
@@ -603,6 +608,17 @@ private String generateEvent(String descTemp, SkriptEventInfo<?> info, @Nullable
 		String[] keywords = info.getKeywords();
 		desc = desc.replace("${element.keywords}", keywords == null ? "" : Joiner.on(", ").join(keywords));
 
+		// Cancellable
+		boolean cancellable = false;
+		for (Class<? extends Event> event : info.events) {
+			if (Cancellable.class.isAssignableFrom(event)) {
+				cancellable = true; // let's assume all are cancellable otherwise EffCancelEvent would do the rest in action
+				break;
+			}
+		}
+		desc = handleIf(desc, "${if cancellable}", cancellable);
+		desc = desc.replace("${element.cancellable}", !cancellable ? "" : "Yes");
```

Comment:
It seems odd to either show "Cancellable    Yes" or nothing
maybe just always show Cancellable for an event and just mark "Yes" or "No"?

Reply:
Oh 😬 yea I remember I added `No` idk what happened there haha

Reply:
is there any point in adding cancellable no? I feel like it would be beterr to just show `Cancellable` only if it's well cancellable

Reply:
Ops, thank you Fuze! I replied too quick and forgot that I actually made it No first then thought this would make a not much useful info when it's a No hence why the line before will not show this section if `cancellable` is false